build.gradle

```diff
@@ -0,0 +1,98 @@
+plugins {
+    id 'java-gradle-plugin'
+    id 'groovy'
+    id "com.gradle.plugin-publish" version "0.10.0"
+    id "com.github.hierynomus.license" version "0.14.0"
+}
+
+group = 'org.moditect'
+version = '1.0.0-SNAPSHOT'
+
+repositories {
+    mavenCentral()
+    jcenter()
+    gradlePluginPortal()
+    flatDir {
+        dirs "${rootProject.projectDir}/lib"
```

Comment:
Setup a composite build https://docs.gradle.org/current/userguide/composite_builds.html to avoid locally built snapshots

Reply:
Huh just remember that moditect core is built with Maven so composite build is not an option.

Reply:
I will fetch the moditect artifact from jitpack.